charts/lagoon-remote/Chart.yaml

```diff
@@ -18,7 +18,7 @@ type: application
 # This is the chart version. This version number should be incremented each
 # time you make changes to the chart and its templates, including the app
 # version.
-version: 0.10.0
+version: 0.10.1
```

Comment:
This isn't required since there's no longer any change to the chart.

Reply:
It was needed when I had a test in there, but not anymore